suricata/update/parsers.py

```diff
@@ -0,0 +1,240 @@
+
+        (("--disable-conf",),
+            {'metavar': '<filename>',
+                'help': "Filename of rule disable filters"}),
+        (("--enable-conf",),
+            {'metavar': '<filename>',
+                'help': "Filename of rule enable filters"}),
+        (("--modify-conf",),
+            {'metavar': '<filename>',
+                'help': "Filename of rule modification filters"}),
+        (("--drop-conf",),
+            {'metavar': '<filename>',
+                'help': "Filename of drop rule filters"}),
+
+        (("--ignore",),
+            {'metavar': '<pattern>', 'action': 'append', 'default': None,
+                'help': "Filenames to ignore "
+                "(can be specified multiple times; default: *deleted.rules)"}),
+        (("--no-ignore",),
+            {'action': 'store_true', 'default': False,
+                'help': "Disables the ignore option."}),
+        (("--threshold-in",),
+            {'metavar': '<filename>',
+                'help': "Filename of rule thresholding configuration"}),
+        (("--threshold-out",),
+            {'metavar': '<filename>',
+                'help': "Output of processed threshold configuration"}),
+        (("--dump-sample-configs",),
+            {'action': 'store_true', 'default': False,
+                'help': "Dump sample config files to current directory"}),
+        (("--etopen",),
+            {'action': 'store_true',
+                'help': "Use ET-Open rules (default)"}),
+        (("--reload-command",),
+            {'metavar': '<command>',
+                'help': "Command to run after update if modified"}),
+        (("--no-reload",),
+            {'action': 'store_true', 'default': False,
+                'help': "Disable reload"}),
+        (("-T", "--test-command"),
+            {'metavar': '<command>',
+                'help': "Command to test Suricata configuration"}),
+        (("--no-test",),
+            {'action': 'store_true', 'default': False,
+                'help': "Disable testing rules with Suricata"}),
+        (("--no-merge",),
+            {'action': 'store_true', 'default': False,
+                'help': "Do not merge the rules into a single file"}),
+
+        # Hidden argument, --now to bypass the timebased bypass of
+        # updating a ruleset.
+        (("--now",),
+            {'default': False, 'action': 'store_true',
+                'help': argparse.SUPPRESS}),
+
+        # The Python 2.7 argparse module does prefix matching which can be
+        # undesirable. Reserve some names here that would match existing
+        # options to prevent prefix matching.
+        (("--disable",),
+            {'default': False, 'help': argparse.SUPPRESS}),
+        (("--enable",),
+            {'default': False, 'help': argparse.SUPPRESS}),
+        (("--modify",),
+            {'default': False, 'help': argparse.SUPPRESS}),
+        (("--drop",),
+            {'default': False, 'help': argparse.SUPPRESS})
```

Comment:
Could we move these out of the function?

Reply:
Sure. Will create a new PR with the changes requested.
Thanks for reviewing :)